$MirOS: ports/multimedia/livestream/patches/patch-liveMedia_H263plusVideoRTPSink_cpp,v 1.1.7.1 2005/03/18 15:50:47 tg Exp $
--- liveMedia/H263plusVideoRTPSink.cpp.orig	Fri Dec 23 13:00:41 2005
+++ liveMedia/H263plusVideoRTPSink.cpp	Wed Dec 28 18:02:48 2005
@@ -18,6 +18,7 @@ along with this library; if not, write t
 // RTP sink for H.263+ video (RFC 2429)
 // Implementation
 
+#include <stddef.h>
 #include "H263plusVideoRTPSink.hh"
 
 H263plusVideoRTPSink
@@ -65,7 +66,7 @@ void H263plusVideoRTPSink
     }
     if (frameStart[0] != 0 || frameStart[1] != 0) {
       envir() << "H263plusVideoRTPSink::doSpecialFrameHandling(): unexpected non-zero first two bytes: "
-	      << (void*)(frameStart[0]) << "," << (void*)(frameStart[1]) << "\n";
+	      << (void*)(ptrdiff_t)(frameStart[0]) << "," << (void*)(ptrdiff_t)(frameStart[1]) << "\n";
     }
     frameStart[0] = specialHeader>>8;
     frameStart[1] = (unsigned char)specialHeader;
